docs/source/topics/processes/index.rst

```diff
@@ -0,0 +1,29 @@
+.. _topics:engine:processes:
+
+*********
+Processes
+*********
+
+Before you start working with processes, make sure you have read and understood the :ref:`basic concept<topics:processes:concepts>`.
```

Comment:
I don't think this is in the spirit of a background section that we imply that users need to have read to be able to work with processes.

I am also not a fan of this slightly aggressive language that is telling users what to do. I think it's better to use slightly more friendly language that is providing hints and suggestions instead or simply states facts like this:
```suggestion
This section assumes knowledge of the :ref:`basic concepts<topics:processes:concepts>` .
```
We could also add this as a `note::`.